jvm/ml4ir-inference/src/test/integration/ml4ir/inference/tensorflow/TensorFlowInferenceIT.scala

```diff
@@ -0,0 +1,109 @@
+package ml4ir.inference.tensorflow
+
+import scala.collection.JavaConverters._
+import ml4ir.inference.tensorflow.data.{ModelFeaturesConfig, StringMapExampleBuilder, TestData}
+import org.junit.Test
+import org.junit.Assert._
+import org.tensorflow.example._
+
+import scala.collection.mutable.ListBuffer
+import scala.io.Source
+
+@Test
+class TensorFlowInferenceIT extends TestData {
+  val classLoader = getClass.getClassLoader
+
+  def resourceFor(path: String) = classLoader.getResource(path).getPath
+
+  /**
+   * Class holding a Example and prediction from Python
+   * @param sequence the Example context
+   * @param scores the prediction scores
+   */
+  class PredictionVector(var sequence: Map[String, String], var scores: Array[Float])
+
+  /**
+   * Ugly parsing of the Python CSV predicition.
+   *
+   * Mainly removing the "b'" formatting and slicing into List and Map.
+   *
+   * @param line
+   * @return
+   */
+  def extractColumnValues(line: String): PredictionVector = {
```

Comment:
change the method signature to be `Option[PredictionVector]`, then when you get number parsing exceptions, you can return None, otherwise return `Some(PredictionVector(sequence, scores))`